### ISSUE TITLE
After uninstalling the program, no website opens

### ISSUE THREAD

**Issue by kerlk** [nomoresat/DPITunnel-android#9]
I tried to delete the program and my phone stopped opening websites. If you reinstall the program, the problem disappears. How do I remove this?

**Comment by kabanito228** [comment]
Yes i see this yesterday, i don't sure, but you can try to delete proxy settings on android settings, or reinstall app and click "restart".

**Comment by jaffar02** [comment]
Happened to me too and I panicked. But after reinstalling app and starting server internet started working, then I stopped server and uninstalled it and now everything working fine
